### PR TITLE
Added a notice to the top of all handbook pages.

### PR DESCRIPTION
**Pull request by DanielRosenwasser**
Instead of deleting them, we can redirect users until we figure out what we want to do.

### CODE REVIEW

**Comment by mhegazy** [comment]
So why not just delete them and replace the links in the main page and in _sidebar to point to the handbook. that is less mental overhead for visitors.

**Comment by DanielRosenwasser** [comment]
Wasn't sure if we were set on deleting them. Sounds good to me.

**Comment by mhegazy** [comment]
let's do it :facepunch: !

**Comment by DanielRosenwasser** [comment]
:facepunch:

File: _Sidebar.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-* [[Generics]]
+* [Basic Types](https://github.com/Microsoft/TypeScript-Handbook/blob/master/pages/Basic%20Types.md)
+* [Interfaces](https://github.com/Microsoft/TypeScript-Handbook/blob/master/pages/Interfaces.md)
+* [Classes](https://github.com/Microsoft/TypeScript-Handbook/blob/master/pages/Classes.md)
+* [Namespaces and Modules](https://github.com/Microsoft/TypeScript-Handbook/blob/master/pages/Namespaces%20and%20Modules.md)
+* [Functions](https://github.com/Microsoft/TypeScript-Handbook/blob/master/pages/Functions.md)
+* [Generics](https://github.com/Microsoft/TypeScript-Handbook/blob/master/pages/Generics.md)
 * [[Common Errors]]
 * [[Integrating with Build Tools]]
 * [[Compiler Options]]
```
Comment (mhegazy) [inline _Sidebar.md]: I wonder if we should move these as well. they are not that different from writing definition files bellow.

Comment (mhegazy) [inline _Sidebar.md]: I would say everything related to the language/toolset goes to the handbook. things related to contributing, releases, compiler API stays in the wiki

**Pull request merged**